examples/simulation-tensorflow-tabular/sim.py

```diff
@@ -0,0 +1,249 @@
+    """Return an evaluation function for server-side (i.e. centralised) evaluation."""
+
+    # The `evaluate` function will be called after every round by the strategy
+    def evaluate(
+        server_round: int,
+        parameters: fl.common.NDArrays,
+        config: Dict[str, fl.common.Scalar],
+    ):
+        # Extract the number of features
+        element_spec = testset.element_spec
+        num_features = element_spec[0].shape[1]
+
+        model = get_model(num_features)  # Construct the model
+        model.set_weights(parameters)  # Update model with the latest parameters
+        loss, accuracy = model.evaluate(testset, verbose=VERBOSE)
+        return loss, {"accuracy": accuracy}
+
+    return evaluate
+
+# Preprocess the data
+def preprocess_data(data):
+    """Preprocess the dataset."""
+    # Select relevant features
+    features = data[RELEVANT_FEATURES].copy()
+    
+    # Convert 'Sex' to binary values
+    features['Sex'] = features['Sex'].map({'male': 0, 'female': 1})
+    
+    # Fill missing 'Age' values with the mean age
+    val_miss_age = features['Age'].mean()
+    features['Age'].fillna(val_miss_age, inplace=True)
+    
+    # Scale the features
+    scaler = StandardScaler()
+    features = pd.DataFrame(scaler.fit_transform(features), columns=RELEVANT_FEATURES)
+
+    data = pd.concat([features,data['Survived']], axis=1)
+    
+    return data
+
+# Load dataset from Hugging Face
+titanic_cds = load_dataset("julien-c/titanic-survival")
+
+# Convert to pandas to preprocess
+titanic_cds = titanic_cds['train'].to_pandas()
+titanic_cds = preprocess_data(titanic_cds)
+
+# Split the data into training and testing sets
+Xy_train, Xy_test = train_test_split(titanic_cds, test_size=0.2, random_state=RANDOM_STATE)
+
+# Set train and test data in Dataset format
+train_cds = {"features": Xy_train[RELEVANT_FEATURES].values.tolist(), "labels": Xy_train["Survived"].values.tolist()}
+train_cds = pd.DataFrame(train_cds)
+train_cds = Dataset.from_pandas(train_cds)
+
+centralized_testset = {"features": Xy_test[RELEVANT_FEATURES].values.tolist(), "labels": Xy_test["Survived"].values.tolist()}
+centralized_testset = pd.DataFrame(centralized_testset)
+centralized_testset = Dataset.from_pandas(centralized_testset).to_tf_dataset(
+    columns="features", label_cols="labels", batch_size=64
+)
+
+# Set partitioner to federate data
+partitioner = DirichletPartitioner(num_partitions=NUM_CLIENTS, partition_by="labels", alpha=ALPHA)
+partitioner.dataset = train_cds
+
+# Create FedAvg strategy
+strategy = fl.server.strategy.FedAvg(
+    fraction_fit=1.0,  # Sample 100% of available clients for training
+    fraction_evaluate=0.5,  # Sample 50% of available clients for evaluation
+    min_fit_clients=2,  # Never sample less than 2 clients for training
+    min_evaluate_clients=2,  # Never sample less than 2 clients for evaluation
+    min_available_clients=int(
+        NUM_CLIENTS * 0.75
+    ),  # Wait until at least 75 clients are available
```

Comment:
not 75 but int(5 * 0.75), not sure if that's intended since there's diff between the code and the comment

Reply:
Thanks! I changed it to 3 to be congruent.

PS: Check the new [PR #3633](https://github.com/adap/flower/pull/3633) with all the modifications.